src/mem/alloc.h

```diff
@@ -45,6 +45,17 @@ namespace snmalloc
     PMNotOurs = 0,
     PMSuperslab = 1,
     PMMediumslab = 2
+
```

Comment:
This description is correct and useful.  Thank you. Could you remove the first person supposition :-)